concepts/05 UI Components/Gantt/20 Filter Data/1 Filter Row.md

```diff
@@ -0,0 +1,407 @@
+    import 'devexpress-gantt/dist/dx-gantt.css'; 
+    
+    import {
+      DxGantt,
+      DxFilterRow,
+      DxColumn,
+      // ...
+    } from 'devextreme-vue/gantt';
+
+    export default {
+      components: {
+        DxGantt,
+        DxFilterRow,
+        DxColumn,
+        // ...
+      }
+    }
+    </script>
+
+##### React
+
+    <!-- tab: App.js -->
+    import React from 'react';
+    import 'devextreme/dist/css/dx.light.css';
+    import 'devexpress-gantt/dist/dx-gantt.css'; 
+
+    import Gantt, {
+        FilterRow,
+        Column,
+        // ...
+    } from 'devextreme-react/gantt';
+
+    const App = () => {
+        return (
+            <Gantt ... >
+                <FilterRow
+                    visible={true}
+                />
+                <Column 
+                    dataField="title"
+                    caption="Subject"
+                    filterValue="Scope"
+                />
+                <Column
+                    dataField="start"
+                    caption="Start Date"
+                    allowFiltering={false} 
+                />
+                {/* ... */}
+            </Gantt>
+        );
+    };
+
+   export default App;
+
+##### ASP.NET Core Controls
+
+    <!--Razor C#-->
+    @(Html.DevExtreme().Gantt()
+        .Columns(columns => {
+            columns.AddFor(m => m.Title)
+                .DataField("title")
+                .Caption("Subject")
+                .FilterValue("Scope");
+            columns.AddFor(m => m.Title)
+                .DataField("start")
+                .Caption("Start Date")
+                .AllowFiltering(false);
+            @* ... *@
+        })
+        .FilterRow(e => {
+            e.Visible(true);
+        })
+        // ...
+    )
+
+##### ASP.NET MVC Controls
+
+    <!--Razor C#-->
+    @(Html.DevExtreme().Gantt()
+        .Columns(columns => {
+            columns.AddFor(m => m.Title)
+                .DataField("title")
+                .Caption("Subject")
+                .FilterValue("Scope");
+            columns.AddFor(m => m.Title)
+                .DataField("start")
+                .Caption("Start Date")
+                .AllowFiltering(false);
+            @* ... *@
+        })
+        .FilterRow(e => {
+            e.Visible(true);
+        })
+        // ...
+    )
+
+---
+
+The **Gantt** supports a predefined set of filter operations for each data type. 
+
+Underlying [dataType](/Documentation/ApiReference/UI_Components/dxGantt/Configuration/columns/#dataType) defines editor types used within filter row cells:
+
+{|
+|-
+! **dataType**
+! **Editor**
+|-
+
+| *"string", "number", "object"*
+| Text box
+|-
+
+| *"boolean"*
+| Drop-down list 
+|-
+
+| *"date"*
+| Date picker
+|-
+
+| *"datetime"*
+| Date and time picker 
+|-
+|}
+
+You can specify necessary filter operations for each column with the [filterOperations](/Documentation/ApiReference/UI_Components/dxGantt/Configuration/columns/#filterOperations) property.
```

Comment:
Hmm... This is a somewhat strange preposition choice: "to **specify** something **with** the X..."

Did you mean something like "assign a necessary combination of filter operation to the `filterOperations` property for each column"?

Reply:
Please, do be careful with "with", for it often adds vagueness in otherwise clear sentences.